compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/DeclarationOrigins.kt

```diff
@@ -45,5 +45,6 @@ interface JvmLoweredDeclarationOrigin : IrDeclarationOrigin {
     object GENERATED_ASSERTION_ENABLED_FIELD : IrDeclarationOriginImpl("GENERATED_ASSERTION_ENABLED_FIELD", isSynthetic = true)
     object GENERATED_MAIN_FOR_PARAMETERLESS_MAIN_METHOD : IrDeclarationOriginImpl("GENERATED_MAIN_FOR_PARAMETERLESS_MAIN_METHOD", isSynthetic = true)
     object SUSPEND_FUNCTION_VIEW : IrDeclarationOriginImpl("SUSPEND_FUNCTION_VIEW")
-    object SUSPEND_IMPL_STATIC_FUNCTION :  IrDeclarationOriginImpl("SUSPEND_IMPL_STATIC_FUNCTION", isSynthetic = true)
+    object SUSPEND_IMPL_STATIC_FUNCTION : IrDeclarationOriginImpl("SUSPEND_IMPL_STATIC_FUNCTION", isSynthetic = true)
+    object INTERFACE_COMPANION_INSTANCE : IrDeclarationOriginImpl("INTERFACE_COMPANION_INSTANCE", isSynthetic = true)
```

Comment:
INTERFACE_COMPANION_INTERMEDIATE/PRIVATE?_INSTANCE  - can't suggest smth better, any ideas?

Reply:
Yeah, same problem here. ..._PRIVATE_... has at least some meaning here and is consistent with the JvmDeclarationFactory method name, so I'll use that one.